themes/custom/az_barrio/css/ckeditor5.css

```
@@ -3,6 +3,14 @@
   flex-wrap: wrap;
 }
 
+/**
+ * Fix source edit wrap issue
+ * See https://github.com/az-digital/az_quickstart/issues/2868.
+ */
+.ck-source-editing-area {
```

Long lines are not wrapped in the standard editor window as well: could we select that <div> as well?

```suggestion
.ck.ck-content, .ck-source-editing-area {
```

---

This would be preferred.